src/msgs/transmsgs.de_DE2.sql

```diff
@@ -2,33 +2,33 @@ set bulk_insert INSERT INTO TRANSMSGS (ENG_TEXT, ENG_ACTION, ENG_EXPLANATION, FA
 -- Notice LOCALE is hardcoded as 'de_DE' in the INSERT statement and therefore the rows below have one fields less than the table.
 -- JRD
 ('', NULL, NULL, 0, 0, '', NULL, NULL, 'daves', '1993-07-16 12:15:54')
-('arithmetic exception, numeric overflow, or string truncation', NULL, NULL, 0, 1, 'Arithmetischer Fehler, numerischer �berlauf oder Zeichenkette abgeschnitten', NULL, NULL, 'daves', '1993-07-16 12:16:07')
```

Comment:
These changes seem to have corrupted the character encoding of the file.

Reply:
Fixed.